deploy/backplane/mtsre/30-mtsre-credentials-request-readers.SubjectPermission.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-mtsre-credentials-request-readers
+  namespace: openshift-rbac-permissions
+spec:
+  permissions:
+  - allowFirst: true
+    clusterRoleName: system:openshift:cloud-credential-operator:cluster-reader
+    namespacesAllowedRegex: (^redhat-.*)
```

Comment:
We'd better list out the namespaces here and update it when needed.

Reply:
As mentioned [here](https://github.com/openshift/managed-cluster-config/pull/1084#issuecomment-1067731927), I'd suggest to keep it `redhat-.*` only because access to the `CredentialsRequest` CR (at least read-only access) would be potentially required in other namespaces as well, considering that any other addon could depend on CRO in a different namespace.

wdyt :)

Reply:
There are some addons that maintained by other teams and namespaces prefixed with redhat-* as well, so we'd better list what NS that your team is managing.

Reply:
Being the MT-SRE team, we're supposed to manage/SRE for all the addons out there.

So, any namespace where any addon could be installed i.e. `redhat-.*` comes under what we are supposed to deal with :)

Reply:
Alright then, let me migrate this `SubjectPermission` under `mtsre/managed-odh` with namespaces as (^redhat-ods-applications$)

Reply:
Hi @cblecker, do you mind checking the above comment out in case you missed it :)
Thanks!

Reply:
@yashvardhan-kukreja Adding specific permissions for new addons is already required, as outlined in the subdirectories of https://github.com/openshift/managed-cluster-config/tree/master/deploy/backplane/mtsre

On further thought, this likely should just get added to the specific subject permissions file for that addon.

Reply:
Alright, @cblecker . Updated the PR with that :)